src/common/Card/helper.ts

```diff
@@ -0,0 +1,41 @@
+// just a list of all months in order
+const months = [
+  "January",
+  "February",
+  "March",
+  "April",
+  "May",
+  "June",
+  "July",
+  "August",
+  "September",
+  "October",
+  "November",
+  "December",
+];
+
+// Input: Date object, duration (minutes)
+// Output: Formatted string for start time and end time of an event
+export const format_time = (time: Date, duration: number): string => {
+  // Input: Date object
+  // Output: String in format HH:MM(AM/PM)
+  const stringify = (time: Date): string => {
```

Comment:
This is a nifty function to have, but I feel like using (moments.js)[https://momentjs.com/] would be nicer. It has easy formatting functions to output dates as strings

Reply:
implemented moment.js, thank you!